lib/project-setup.js

```diff
@@ -0,0 +1,66 @@
+/**
+ * Copyright (c) 2017, Kinvey, Inc. All rights reserved.
+ *
+ * This software is licensed to you under the Kinvey terms of service located at
+ * http://www.kinvey.com/terms-of-use. By downloading, accessing and/or using this
+ * software, you hereby accept such terms of service  (and any agreement referenced
+ * therein) and agree that you have read, understand and agree to be bound by such
+ * terms of service and are of legal age to agree to such terms with Kinvey.
+ *
+ * This software contains valuable confidential and proprietary information of
+ * KINVEY, INC and is subject to applicable licensing agreements.
+ * Unauthorized reproduction, transmission or distribution of this file and its
+ * contents is a violation of applicable laws.
+ */
+
+const { isEmpty, isNullOrUndefined, readJSONSync, writeJSON } = require('./utils');
+
+class ProjectSetup {
+  constructor(path) {
+    this._path = path;
+    this._flexNamespace = {};
+  }
+
+  save(done) {
+    const data = {
+      flex: this._flexNamespace
+    };
+
+    writeJSON(this._path, data, done);
+  }
+
+  load() {
+    try {
+      const projectSetup = readJSONSync(this._path);
+      if (!isEmpty(projectSetup)) {
+        this._flexNamespace = projectSetup.flex;
+      }
+    } catch (ex) {
+      return ex;
+    }
+  }
+
+  isFlexSet() {
```

Comment:
Which part of Flex does this refer to? Setup/config/service ID/etc. (ambiguous name)